### PR TITLE
updated create order using api

### PR DESCRIPTION
**Pull request by ghada-37**
I need your help, I fixed payload of create order API 

to run it use command :
`nosetests -vs --nologcapture --tc-file=config.ini ui_testing/testcases/basic_tests/test06_orders.py:OrdersTestCases.test036_Duplicate_sub_order_with_multiple_testplans_and_testunits_delet_approach
`

but I still have status = 0 and api meesage :
`{'status': 0, 'message': "Cannot read property 'id' of undefined"}
`

### CODE REVIEW

**Comment by 0xIslamTaha** [comment]
@ghada-37 I'm trying to debug it now

**Comment by 0xIslamTaha** [comment]
@ghada-37 Finally, create_order is working 
image


To create an order with two test unit, I created this [method](https://github.com/modeso-open-source/1lims-automation/blob/c48b796607d1d8575780fbde4d5a6db18dfe46f9/api_testing/apis/orders_api.py#L241) but u may select an artical which doesnt have another complete test plan so I need to think more about the logic here.

File: api_testing/apis/orders_api.py [hunk 1]
```diff
@@ -48,45 +49,59 @@ def create_new_order(self, **kwargs):
...
+            testplan = random.choice(TestPlanAPI().get_completed_testplans())
+            material_type = testplan['materialType']
+            material_type_id = GeneralUtilitiesAPI().get_material_id(material_type)
+            article = testplan['article'][0]
+            article_api = ArticleAPI()
+            if article == "all":
+                res, _ = article_api.get_all_articles(limit=20)
+            else:
+                res, _ = article_api.quick_search_article(name=article)
+            article_id = res['articles'][0]['id']
```
Comment (ghada-37) [inline api_testing/apis/orders_api.py:64]: article_id is already in test plan
`article_id = testplan['articleNo'][0]`

Comment (0xIslamTaha) [reply api_testing/apis/orders_api.py:64]: @ghada-37 no it's not

**Comment by 0xIslamTaha** [comment]
@ghada-37 I created another branch from this one and merged only the API fix to master

File: api_testing/apis/orders_api.py (hunk 1 above)
Comment (ghada-37) [reply api_testing/apis/orders_api.py:64]: ok, thanks

**Pull request closed without merging**